### PR TITLE
Add Mishipay case study

### LINKED ISSUE #7410: [Fugu][PWA] MishiPay Shape Detection API Case Study [3/31/2022]
* Proposed title: MishiPay Shape Detection API Case Study
* Proposed by: tsteiner
* Related areas: Fugu/Capabilities,PWA
* Content type: Case Study (https://web.dev/handbook/case-study-guidelines/)
* Deadline: 3/31/2022
* Deadline explanation: None really.
* [Draft](https://docs.google.com/document/d/1mbK9zqzkMAtEUHKilBhqZP6XZeIRsuZmCGsQXYIKlkg/edit?usp=sharing)
* Content summary: Web app is faster to load than native app and does barcode detection for easy checkout.

### PR DESCRIPTION
**Pull request by tomayac**
Fixes #7410 

Changes proposed in this pull request:

- Add the case study.

### CODE REVIEW

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:211]:
```suggestion
acceleration, WebGL API and its
[`WebGL2RenderingContext`](https://developer.mozilla.org/docs/Web/API/WebGL2RenderingContext) can optimize gains on the CPU-intensive pre-processing tasks.
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:215]:
```suggestion
The final layer is the decoder layer which is responsible for decoding barcodes from the frames
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:218]:
```suggestion
[Shape Detection API](https://developer.mozilla.org/docs/Web/API/Barcode_Detection_API)
(which is not yet available on all browsers) the browser itself decodes the barcode
```

Comment (jpmedley) [reply src/site/content/en/blog/mishipay/index.md:218]: >which is not yet available...

Just a bit of future proofing.

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:258]:
```suggestion
For devices that don't support the Shape Detection API yet, we need a fallback solution to
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:262]:
```suggestion
method which helps switch between the Shape
Detection API and the fallback solution.
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:266]:
```suggestion
BarcodeDetector.getSupportedFormats()
.then((supportedFormats) => {
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:269]: I can't call a function that doesn't exist on my browser. It looks like you're missing the feature detection code that illustrates the very point of the preceding paragraph.

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:271]: Something went wrong with this sentence. I don't know what it's telling me.

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:276]:
```suggestion
with any web application to implement scanning. Here are some of the libraries that we
```

Comment (jpmedley) [reply src/site/content/en/blog/mishipay/index.md:276]: Who is 'we' in this sentence? Until now, it has been MishiPay, but it will be taken as Google. Has Google exhaustively tested these? How about 'Here are some widely used libraries.'

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:321]:
```suggestion
All the above libraries are full-fledged SDKs that compose all the layers discussed above. They
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:322]:
```suggestion
also expose interfaces to support various scanning operations. Depending on the barcode formats
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:323]: Are you saying that some barcode needs may prevent me from using one of the WASM solutions? This sentence isn't clear.

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:327]:
```suggestion
[Scandit](https://docs.scandit.com/stable/web/) was our primary choice. It supports all barcode
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:337]: Who believes this? MishiPay or Google?

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:342]:
```suggestion
to understand the value that an application gives them before they download it.
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:343]:
```suggestion
In a store, where MishiPay saves shoppers' time and improves their
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:344]:
```suggestion
experience, it is counterintuitive to wait for a download
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:345]:
```suggestion
before they can use an application. This is where our PWA helps.
```

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [inline src/site/content/en/blog/mishipay/index.md:346]:
```suggestion
By eliminating the barrier to entry, we have increased our transactions by 10 times and enabled our users
```

**Review by jpmedley — CHANGES_REQUESTED** [review]

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:48]:
```suggestion
Users find our technology extremely helpful when waiting in a queue or check-out line, as it allows them to skip
```

Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:48]: Extremely helpful sounds good to me instead.

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:49]:
```suggestion
the queue and have a smooth in-store experience. But the hassle of downloading an Android or iOS
```

Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:49]: Smooth instead.

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:51]:
```suggestion
MishiPay, and we needed to increase user adoption with a lower barrier of entry.
```

Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:51]: Lower barrier of entry instead.

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:92]: Occasionally

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:132]: Yeah, right. Switched to "core feature".

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:269]: Added feature detection.

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:271]:
```suggestion
{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/oYmpkpsdwmtA8e440F9I.png", alt="Flow diagram showing how, dependent on Barcode Detector support and the supported barcode formats, either the Shape Detection API or the fallback solution  is being used.", width="800", height="404" %}
```

Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:271]: Rephrased.

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:276]: We → MishiPay

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:323]: Not all barcode formats are supported by all libraries.

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:334]:
```suggestion
new HTML element `<scanner>` that has the capabilities required for a barcode scanner. Engineering at MishiPay believes
```

Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:334]: MishiPay

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:330]:
```suggestion
and detection speed needed for the business case, a decision can be between Wasm and non-Wasm solutions. Despite the
```

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:88]: Moved around.

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md]: MishiPay

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:345]:
```suggestion
to save 2.5 years of waiting in the queue.

## Acknowledgements

This article was reviewed by
[Joe Medley](https://github.com/jpmedley).
```

**Comment by tomayac** [comment]
@jpmedley PTAL. 

Added `DO NOT TRACK` label since we're waiting for MishiPay's final approval before publishing.

File: src/site/content/en/blog/mishipay/index.md
Comment (jpmedley) [reply src/site/content/en/blog/mishipay/index.md:275]: We typically wrap the `then()` clause (if we're not using `await`. Aside from consistency, this avoids horizontal scrolling on small screens.

Comment (tomayac) [reply src/site/content/en/blog/mishipay/index.md:275]: Sorry, this was prettier. Overruled it manually.

**Review by jpmedley — APPROVED** [review]

File: src/site/_data/authorsData.json
Comment (tomayac) [inline src/site/_data/authorsData.json:1276]:
```suggestion
    "twitter": "nikil_mathew",
    "linkedin": "https://www.linkedin.com/in/nikilmat/",
```

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:15]:
```suggestion
hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/G1Q7oGsqt7t33ctT4de2.png
```

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:281]:
```suggestion
{% Img src="image/8WbTDNrhLsU0El80frMBGE4eMCD3/GmrPC7mFTdtsEdS1BDWp.png", alt="Flow diagram showing how, dependent on Barcode Detector support and the supported barcode formats, either the Shape Detection API or the fallback solution  is being used.", width="800", height="404" %}
```

File: src/site/content/en/blog/mishipay/index.md
Comment (tomayac) [inline src/site/content/en/blog/mishipay/index.md:15]:
```suggestion
hero: image/8WbTDNrhLsU0El80frMBGE4eMCD3/CZo4R87iOBYiRpIq6NcP.jpg
```

**Pull request merged**
